packages/desktop-client/src/components/manager/ConfigServer.tsx

```diff
@@ -77,15 +77,15 @@ export default function ConfigServer() {
   }
 
   async function onSkip() {
-    await setServerUrl(null);
+    await setServerUrl('');
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏we need to keep `null` here.

https://github.com/actualbudget/actual/blob/a8a0f777e2369a9f74becabab9f1eb96d50cba16/packages/loot-core/src/server/main.ts#L1775